test/models/user_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  
+  def setup
+    foo = ('a'..'z').to_a.shuffle.first(8).join # random 8char string for unique email
+    @user = User.new({:email => "#{foo}@gmail.com", :first_name => "someFirstName", 
```

Comment:
Line is too long. [84/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Trailing whitespace detected.